src/reportabug/__init__.py

```diff
@@ -192,7 +192,13 @@ def format_markdown(data, github=False):
     )
     print()
     print("* Python", data["platform"]["version"], data["platform"]["architecture"])
-    print("* Platform", data["platform"]["platform"], data["platform"]["machine"])
+
+    platform_info = (data["platform"]["platform"]
+                     if data["platform"]["machine"] in
+                     data["platform"]["platform"] else
+                     data["platform"]["platform"] + ' ' + data["platform"]["machine"])
+
+    print("* Platform %s " % (platform_info))
```

Comment:
I think this is more easy to read as a regular `if` block:

```python
if ...machine not in ...platform:
    print('* Platform', both)
else:
    print('* Platform', one)
```

Reply:
Hello  sorry for the delay. I send the commit. :-)